auacm/app/modules/blog_manager/blog_test.py

```diff
@@ -35,59 +33,57 @@ def testCreate(self):
                 self.assertEqual(post[key], post_response[key])
 
         # Delete the post from the test database
-        session.flush()
-        post = session.query(BlogPost).filter(
-            BlogPost.id == post_response['id']).first()
-        session.delete(post)
-        session.commit()
+        DATABASE_SESSION.flush()
+        post = DATABASE_SESSION.query(BlogPost).filter(
+            BlogPost.post_id == post_response['id']).first()
+        DATABASE_SESSION.delete(post)
+        DATABASE_SESSION.commit()
 
-    def testGetAll(self):
```

Comment:
I believe that is correct, but I feel like camel case is not supposed to be used at all in Python. I think we should use the standard `test_` at the beginning of the test method.